src/screens/MainScreen/components/Train/index.js

```diff
@@ -1,23 +1,58 @@
 import React from "react";
 import { Container, TrainImg } from "./styles";
+import { CSSTransition } from "react-transition-group";
 import TrainHeadGif from "../../../../assets/img/train-head.gif";
 import TrainTailGif from "../../../../assets/img/train-tail.gif";
 import Cart from "./components/Cart";
 
-const Train = ({ carts, cartSize }) => (
-  <Container>
-    <TrainImg cartSize={cartSize} src={TrainHeadGif} />
-    {carts.map(({ name, color, timestamp, facepic }) => (
-      <Cart
-        key={timestamp}
-        name={name}
-        color={color}
-        facepic={facepic}
-        cartSize={cartSize}
-      />
-    ))}
-    <TrainImg cartSize={cartSize} src={TrainTailGif} />
-  </Container>
-);
+const Train = ({ carts, cartSize, leaving, animationDuration }) => {
+  return (
+    <Container>
+      <CSSTransition
+        key="train-head"
+        classNames="cart"
+        timeout={animationDuration}
+        in={!leaving}
```

Comment:
the github diff is highlighting `in` as a keyword :0

Reply:
Yeah well that's what they decided to name the prop sooo idk haha